configs/pingcap.json

```diff
@@ -1,8 +1,23 @@
 {
   "index_name": "pingcap",
   "start_urls": [
-    "https://pingcap.com/docs",
-    "https://pingcap.com/"
+    {
+      "url": "https://pingcap.com/docs(-?)(?P<lang>.*?)/",
```

Comment:
This will only crawled chinese documentation in the best case scenario ( (-?) will be add as it within the url and thus making an error)
You should try to keep a consistency within you URL by always adding the language in it (or using at least a redirection)